[tutorials] Working with Lists tutorial (#730)
lit/lit.dev: packages/lit-dev-content/site/tutorials/content/working-with-lists/05.md

When using the `map()` directive or just providing an iterable within the
template, Lit creates a list of DOM nodes and updates them in the iteration
order, efficiently reusing any previously rendered nodes. This is **good** and

> should be utilized when **all states are being controlled by Lit**.

Reviewer: How about: `These techniques should only be used when Lit controls all of the rendered state as is commonly the case.`

> However, when relying instead on the DOM nodes to keep some state that are not

Reviewer: ```suggestion
However, when relying instead on the DOM nodes to maintain state that is not
```

> controlled by Lit, buggy behavior could arise. This is an example of when the

Reviewer: ```suggestion
controlled by Lit, unexpected behavior can occur, as we'll see. In this case, use the `repeat()` directive instead.
```

> `repeat()` directive should be utilized.

Reviewer: ```suggestion

```
